sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/src/Generated/DigitalTwinsDeleteOperation.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.ResourceManager.DigitalTwins
+{
+    /// <summary> Delete a DigitalTwinsInstance. </summary>
+    public partial class DigitalTwinsDeleteOperation : Operation<Response>, IOperationSource<Response>
```

Comment:
Mmm, these operations should have been grouped by the resources not individual REST actions. I need to double check the codegen on this.

Reply:
these are LRO operations. Not operation groups. Will try to address in API redesign.